conda/core/solve.py

```diff
@@ -48,21 +49,13 @@ def _get_solver_class(key=None):
 
     See ``context.solver`` for more details.
     """
-    solvers = context.plugin_manager.get_hook_results("solvers")
-    key = (key or context.solver).lower()
-
-    solvers_mapping = {}
-    for solver in solvers:
-        solvers_mapping[solver.name.lower()] = solver.backend
-
-    if key in solvers_mapping:
-        return solvers_mapping[key]
-    else:
-        raise CondaValueError(
-            f"You have chosen a non-default solver backend ({key}) "
-            f"but it was not recognized. Choose one of: "
-            f"{', '.join(solvers_mapping.keys())}"
-        )
+    warnings.warn(
+        "`conda.core.solve._get_solver_class` is pending deprecation and will be removed in a "
+        "future release. Please use `conda.base.context.plugin_manager.get_cached_solver_backend ",
```

Comment:
```suggestion
        "future release. Please use `conda.base.context.plugin_manager.get_cached_solver_backend "
```

This comma doesn't belong there, right? 😅

Reply:
Yep!